examples/voice-assistant/simple-rag/assistant.py

```diff
@@ -46,7 +46,7 @@ async def _enrich_with_rag(assistant: VoiceAssistant, chat_ctx: llm.ChatContext)
         stt=deepgram.STT(),
         llm=openai.LLM(),
         tts=openai.TTS(),
-        before_llm_cb=_enrich_with_rag,
+        will_synthesize_assistant_reply=_enrich_with_rag,
```

Comment:
`before_llm_cb` is the right parameter. This was renamed in 0.8.11

Reply:
Okay. Apologies, for generating this PR before reviewing the latest version of the livekit. 
Thanks for this.

Reply:
no worries at all!